drivers/Web/test_bankApplication.py

```diff
@@ -26,18 +26,22 @@ def test_request_root_resource_is_html():
     header = {}
     body = b''
     method = "GET"
-    resource = httpRequest.resource("/", {})
+    resource = httpRequest.http_resource("/", {})
     version = "1.1"
     request = httpRequest.httpRequest(header, body, method, resource, version)
     bankApplicationReturn = bankApplication.root(request)
     assert bankApplicationReturn.getHeaders()["Content-Type"] == "text/html"
 
 def test_post_status_200():
     header = {}
-    resource = httpRequest.resource("/", {})
+    resource = httpRequest.http_resource("/", {})
     method = "POST"
     body = b"login=ryanbanco&password=abc123"
     version = "1.1"
     request = httpRequest.httpRequest(header, body, method, resource, version)
     bankApplicationReturn = bankApplication.root(request)
     assert bankApplicationReturn.getStatus() == 200
+
```

Comment:
this code must be removed. Use pytest feature to select a test to run or your IDE